src/sys/wait.rs

```diff
@@ -46,6 +48,30 @@ libc_bitflags!(
     }
 );
 
+/// Possible child targets of`waitpid()`.
+///
+/// The most common variants for waiting are waiting
+/// for a specific child via `WaitPid::Child(pid)` and
+/// waiting for any child via `WaitPid::AnyChild`.
+#[derive(Eq, PartialEq, Clone, Copy, Debug)]
+pub enum WaitTarget {
+    /// Waiting for a child with `Pid` process id.
+    Child(Pid),
+    /// Waiting for a child of the same process group as the calling process.
+    ThisGroupChild,
+    /// Waiting for any child.
+    AnyChild,
+    /// Waiting for a child of process group `Pid` (NB: do not negate this value,
```

Comment:
What does NB here mean?

Reply:
It's [Nota Bene](https://en.wikipedia.org/wiki/Nota_bene). I'll replace it with "please, note" since it adds confusion.